ethcore/engines/ethash/src/lib.rs

```diff
@@ -464,7 +464,18 @@ impl Ethash {
 	}
 }
 
-fn ecip1017_eras_block_reward(era_rounds: u64, mut reward: U256, block_number:u64) -> (u64, U256) {
+
+/// Calculates the number of eras and reward
+///
+/// # Panics
+///
+/// This function will panic if `era_rounds` is less than `2`
+fn ecip1017_eras_block_reward(era_rounds: u64, mut reward: U256, block_number: u64) -> (u64, U256) {
+	// NOTE(niklasad1): all numbers is divisible by 1, it will cause the if below
+	// to succeed except the first block. Thus, `era_rounds - 1 == 0` and cause `divide by zero`
+	//
+	// TODO(niklasad): We could set `eras` to zero if this occurs, but I'm not sure of the implications
```

Comment:
thoughts?

Reply:
I think it's fine to assume blocks in one rounds is at least 2 (it's set to 5_000_000 for ETC if I'm not mistaken).

Reply:
Let's remove the TODO and we're good to merge.

Reply:
the note too?

Reply:
the note is good for future readers